squire/urls.py

```diff
@@ -1,22 +1,11 @@
-"""base URL Configuration
+"""base URL Configuration"""
```

Comment:
Why remove this comment?

Reply:
I don't know, probably should indeed not be removed.